socorro/unittest/processor/test_signature_utilities.py

```diff
@@ -1117,15 +1076,15 @@ def test_action_2(self):
         }
 
         # the call to be tested
-        ok_(sgr._action(raw_crash, {}, processed_crash, processor_meta))
+        assert sgr._action(raw_crash, {}, processed_crash, processor_meta) is True
 
-        eq_(
-            processed_crash['signature'],
+        assert (
+            processed_crash['signature'] ==
```

Comment:
Depending on your editor and how it does syntax highlighting this might be less subtle. I've been doing this for a while so it clicks for me, but using an intermediary variable makes sense and readability and it's stepsisge, maintainability, are important. I'll do that going forward.